src/modules/admissions/components/ApplicantPortal/SetPriorities.tsx

```diff
@@ -78,30 +103,80 @@ export const SetPriorities: React.VFC<SetPrioritiesProps> = ({
   // Saves priorities and moves the applicant to the interview booking phase
   const { patchApplicant } = usePatchApplicant()
 
-  const [deleteInternalGroupPriority] = useMutation(
-    DELETE_INTERNAL_GROUP_POSITION_PRIORITY,
-    { refetchQueries: ['GetApplicantFromToken'] }
-  )
+  const [deleteInternalGroupPriority] = useMutation<
+    ApplicantDeleteInternalGroupPositionPriorityReturns,
+    ApplicantDeleteInternalGroupPositionPriorityVariables
+  >(APPLICANT_DELETE_INTERNAL_GROUP_POSITION_PRIORITY, {
+    refetchQueries: ['GetApplicantFromToken'],
+  })
 
   const [addPriority] = useMutation<
     AddInternalGroupPositionPriorityReturns,
     AddInternalGroupPositionPriorityVariables
-  >(ADD_INTERNAL_GROUP_POSITION_PRIORITY, {
+  >(APPLICANT_ADD_INTERNAL_GROUP_POSITION_PRIORITY, {
     refetchQueries: ['GetApplicantFromToken'],
   })
 
+  const [updatePriorities] = useMutation<
+    UpdateInternalGroupPositionPriorityOrderReturns,
+    ApplicantUpdateInternalGroupPositionPriorityOrderVariables
+  >(APPLICANT_UPDATE_INTERNAL_GROUP_POSITION_PRIORITY_ORDER_MUTATION)
+
   //  === Handlers ===
   const handleAddPriority = (internal_group_position_id: string) => {
     addPriority({
       variables: {
         internalGroupPositionId: internal_group_position_id,
         applicantId: applicant.id,
+        token: applicantToken,
+      },
+    })
+  }
+
+  const handlePriorityChange = (index: number, direction: -1 | 1) => {
+    handlers.reorder({ from: index, to: index + direction })
+    setIsDirty(true)
+  }
+  const handleUpdatePriorities = (priorityIds: string[]) => {
+    console.table(priorityIds)
+    updatePriorities({
+      variables: {
+        priorityOrder: priorityIds,
+        applicantId: applicant.id,
+        token: applicantToken,
+      },
+      onError(message) {
+        showNotification({
+          title: 'Error',
+          message: `${message}`,
```

Comment:
```suggestion
      onError({message}) {
        showNotification({
          title: 'Error',
          message
```

Reply:
I'm guessing that we don't actually want to display the errors to the user via a notification.